content/sigs/advocacy-and-outreach/outreach-programs/index.adoc

```diff
@@ -39,24 +39,23 @@ in completing their summer projects.
 For more information, see link:/projects/gsoc/[our GSoC project page].
 
 To contact us regarding this program,
-send us a message on the
-link:https://groups.google.com/forum/#!forum/jenkinsci-gsoc-all-public[GSoC mailing list]
+send us a message on the link:https://community.jenkins.io/c/contributing/gsoc/6[Jenkins Community Discourse]
 or contact us on the
 link:https://gitter.im/jenkinsci/gsoc-sig[GSoC Gitter channel].
 
 // The GSoC logo is a bit tall, so add some empty lines
 {empty} +
 
-=== Outreachy
+=== She Code Africa Contributhon 
 
-image:/images/outreachy/outreachy_small.png[Outreachy, role=center, float=right]
+image:/images/post-images/2020-03-contributhon/she-code-africa-logo.svg[She Code Africa logo, role=center, float=right]
```

Comment:
If you're willing to consider it, I'd prefer that we set the height of the image so that it does not extend below the end of the paragraph text.  See the preview before the change to get a sense of the change
```suggestion
image:/images/post-images/2020-03-contributhon/she-code-africa-logo.svg[She Code Africa logo, role=center, float=right, height=180]
```